10981.0.txt

```diff
@@ -0,0 +1,38 @@
+@0
+
+
+@1
+MOV UP, RIGHT
+#INPUT.A RIGHT
+
+@2
+MOV LEFT, ACC
+#STORE INPUT IN
+#ACC
```

Comment:
Good effort at cramming meaningful comments into the space allowed in the game -- it's good that you didn't stick to this for later levels though!